webapp/src/app/ui/pull-request-widget/pull-request-widget.component.html

```diff
@@ -0,0 +1,21 @@
+<div class="border border-gray-300 rounded-lg p-4 w-72">
+  <div class="flex justify-between items-center mb-2 text-xs text-gray-500">
+    <span class="font-bold">{{ pullRequest().repository.name }} #{{ pullRequest().number }} on {{ pullRequest().createdAt }}</span>
```

Comment:
Can you als add an artifact icon, e.g.:
https://primer.style/components/icon

Maybe also name it more generic, this could be an entity that we can also use for issues. 

You can reference the leaderboard how to include an icon from octicons